packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -442,20 +446,20 @@ function createApp(
     : ` --interactive=false`;
   const defaultBaseArg = defaultBase ? ` --defaultBase="${defaultBase}"` : ``;
 
+  const executablePath = `${getPackageManagerRun(packageManager)} ${
+    cli.command
+  }`;
   console.log(
-    `new ${name} ${args} --preset="${preset}"${appNameArg}${styleArg}${nxCloudArg}${interactiveArg}${defaultBaseArg} --collection=@nrwl/workspace`
+    `${executablePath} new ${name} ${args} --preset="${preset}"${appNameArg}${styleArg}${nxCloudArg}${interactiveArg}${defaultBaseArg} --collection=@nrwl/workspace`
   );
-  const executablePath = path.join(tmpDir, 'node_modules', '.bin', cli.command);
-  const collectionJsonPath = path.join(
-    tmpDir,
-    'node_modules',
-    '@nrwl',
-    'workspace',
-    'collection.json'
+  const collectionJsonPath = require.resolve(
+    '@nrwl/workspace/collection.json',
+    { paths: [tmpDir] }
   );
   execSync(
-    `"${executablePath}" new ${name} ${args} --preset="${preset}"${appNameArg}${styleArg}${nxCloudArg}${interactiveArg}${defaultBaseArg} --collection=${collectionJsonPath}`,
+    `${executablePath} new ${name} ${args} --preset="${preset}"${appNameArg}${styleArg}${nxCloudArg}${interactiveArg}${defaultBaseArg} --collection=${collectionJsonPath}`,
```

Comment:
Does it mean it will execute: `npm run new ...`? Not sure if this is right.

Reply:
`executablePath` also contains the `command.cli`, so it will execute something like `npm run tao new ...`, but for npm it should be `npx` anyway 😄